### ISSUE TITLE
Add alt tags to buttons in FreeUKGen

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/Coordination#54]
I'm thinking we maybe should have alt text on them that provides a context (e.g. 'copy this URL into your RSS feed for updates').

**Comment by DeniseColbert** [comment]
I have spent ages looking for the buttons in Craft and can _not see them. I think they may actually live on another plane.

**Comment by DeniseColbert** [comment]
Completed by Bev, waiting for sign-off on Monday
